tsl/test/expected/recompress_chunk_segmentwise.out

```diff
@@ -483,7 +483,7 @@ select compressed_chunk_name as compressed_chunk_name_after_recompression from c
 select :'compressed_chunk_name_before_recompression' as before_recompression, :'compressed_chunk_name_after_recompression' as after_recompression;
     before_recompression    |    after_recompression     
 ----------------------------+----------------------------
- compress_hyper_13_14_chunk | compress_hyper_13_15_chunk
+ compress_hyper_13_14_chunk | compress_hyper_13_14_chunk
```

Comment:
Comment above on line 475 needs updating since it is incorrect with this change.